### DISCUSSION TITLE
Clourise tabs

### DISCUSSION THREAD

**Discussion by WillemsenGithub** [doublecmd/doublecmd#1516] [Suggestions]
Well, Clourise tabs 
Is that self explanatory ?
Anyway, in Dark theme (Adwaita Dark) on Linux all tabs look the same
Maybe dim the inactive ones, may help as well
Thanks for amazing work

**Comment by alexx2000** [comment]
Impossible, DC use a system theme (GTK2 or Qt 5/6). So choose a suitable theme.

**Reply by WillemsenGithub** [reply]
Ok, thanks
If impossible to colorize the TABs, would it be possible to highlight/capitalize/colorize the Text in the active tab in any way?

(Global theme affects all apps, the present choice seems to fit most apps without too many errors)

**Comment by WillemsenGithub** [comment]
The way out to manipulate Theme, Tabs, Buttons, Colors:

Using the QT.AppImage 

Tune the chosen Theme with **Kvantummanager** to change QT theme that's applied to application
